lib/const.rb

```diff
@@ -282,6 +280,7 @@
 # CMAKE_LIBRARY_PATH is the build LIBRARY_PATH, as opposed to
 # CMAKE_INSTALL_LIBDIR which defaults to lib or lib64.
 CREW_CMAKE_OPTIONS ||= <<~OPT.chomp
+  -DCMAKE_INSTALL_LIBDIR=#{ARCH_LIB} \
```

Comment:
As I recall this should only be used for certain packages, as this overrides the correct value of CMAKE_INSTALL_LIBDIR as set by GNUInstallDirs

Reply:
I'm not sure-- I have been going around fixing packages that don't use GNUInstallDirs, so I'd rather know which ones need to be fixed.

Reply:
fwiw, libarchive now uses `GNUInstallDirs`: https://github.com/libarchive/libarchive/pull/2676, so we can drop the postinstall patch (because thats how our package file works around it) once a new version is released and we upgrade to it.